backend/src/main.cpp

```diff
@@ -487,6 +495,8 @@ static void HandleAfvEvents(afv_native::ClientEventType eventType, void* data, v
             }
 
             NapiHelpers::callElectron("StationDataReceived", callsign, std::to_string(frequency));
+            MainThreadShared::mApiServer->handleAFVEventForWebsocket(
+                sdk::types::Event::kStationStateUpdated, callsign, frequency);
```

Comment:
Required so connected websocket clients can map a callsign to a frequency when stations are added.